generators/generator-constants.js

```diff
@@ -1,15 +1,25 @@
 'use strict';
 
-// all constants used through out all generators
+// all constants used throughout all generators
+
+const MAIN_DIR = 'src/main/';
+const TEST_DIR = 'src/test/';
+
 const constants = {
     QUESTIONS : 15,
-    RESOURCE_DIR : 'src/main/resources/',
-    WEBAPP_DIR : 'src/main/webapp/',
-    ANGULAR_DIR : 'src/main/webapp/app/',
```

Comment:
Oh now I see what you mean, disregard that last comment. I changed this when I was thinking to put `app` and `content` in as separate constants, but didn't revert that bit cleanly. I will amend the PR when I get to work!